src/app/listnews/listnews.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="featured-topic-newsfeed">
+    <header class="newsfeed-header">
+        <h3>Business and Weather</h3>
+    </header>
+    <section class="newsfeed-body">
+        <a class ="featured-story-newsfeed" href="#">
+            <div class="img-featured-story">
+                <img src="https://cdn.jwplayer.com/v2/media/9rPtxfTy/poster.jpg?w=348&h=260&crop=1" alt="img">
```

Comment:
Substituir o conteúdo hardcoded pelo existente na propriedade relativa à notícia highlighted.